gcovr/tests/simple1/reference/coverage.xml

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" ?>
+<?xml version="" ?>
```

Comment:
To double check, is this correct? 
All other files with changes have the XML version added, this one has it _removed_...